src/chapters/errors-and-debugging/asking-good-questions.rst

```diff
@@ -0,0 +1,42 @@
+Asking Good Questions
+=====================
+
+If employing the strategies outlined in this chapter still don't help you find a bug in your code, you always have other programmers to reach out to.
+
+Whether you are asking a question of your teaching assistant, another student, or a stranger in an online forum, you should first be able to answer the 3 questions discussed in this section.
```

Comment:
Remove "a question of".

Replace "discussed in this section" with "outlined below".